package.json

```diff
@@ -2,38 +2,46 @@
   "name": "web-streams-polyfill",
   "version": "3.1.0",
   "description": "Web Streams, based on the WHATWG spec reference implementation",
-  "main": "dist/polyfill",
-  "browser": "dist/polyfill.min.js",
-  "module": "dist/polyfill.mjs",
-  "types": "dist/types/polyfill.d.ts",
-  "typesVersions": {
-    ">=3.6": {
-      "dist/types/*": [
-        "dist/types/ts3.6/*"
-      ]
-    }
+  "main": "dist/ponyfill.js",
+  "browser": "dist/ponyfill.js",
```

Comment:
Hmm, not sure if it's a good idea to make the `browser` target an ES module. 🤔

But since `main == browser`, I might as well remove the explicit `browser` field, so the module loader or bundler will fall back to the `main` or `module` field. At least, from what I understand of [webpack](https://webpack.js.org/configuration/resolve/#resolvemainfields), [rollup](https://www.npmjs.com/package/@rollup/plugin-node-resolve) and [browser-resolve](https://www.npmjs.com/package/browser-resolve).

Reply:
This might be totally unrelated (cuz u have d.ts instead) but another project that used jsdoc for typing did use mjs instead as their main file extension. vscode was unable to follow/check this file so it didn't help with typing. I made a PR that changed it to .js and behold: you got typing support